chromedriver-bin/update.sh

```diff
@@ -1,22 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Since we are only suggesting the installation of the bdi package, maybe we should keep this script for everyone who does not need the added features provided by the bdi package.

Reply:
The package replaces exactly this update script. The "added features" are multi OS and browser support and some command line options